GA05MOAS-PG578/GA05MOAS-PG578_R00001_ingest.csv

```diff
@@ -1,8 +1,8 @@
-parser,filename_mask,reference_designator,data_source,status,notes
-#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,,GA05MOAS-PG578-00-ENG000000,recovered_host,Not Expected,Glider lost
-#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,,GA05MOAS-PG578-01-CTDGVM000,recovered_host,Not Expected,Glider lost
-#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,,GA05MOAS-PG578-02-DOSTAM000,recovered_host,Not Expected,Glider lost
-#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,,GA05MOAS-PG578-03-FLORTM000,recovered_host,Not Expected,Glider lost
-#mi.dataset.driver.moas.gl.flort_o.flort_o_glider_recovered_driver,,GA05MOAS-PG578-04-FLORTO000,recovered_host,Not Expected,Glider lost
-#mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,,GA05MOAS-PG578-05-NUTNRM000,recovered_host,Not Expected,Glider lost
-#mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,,GA05MOAS-PG578-06-PARADM000,recovered_host,Not Expected,Glider lost
\ No newline at end of file
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-00-ENG000000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-01-CTDGVM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-02-DOSTAM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-03-FLORTM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.flort_o.flort_o_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-04-FLORTO000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-05-NUTNRM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-06-PARADM000,recovered_host,Not Expected,Glider lost
```

Comment:
Looks good... 
Does NUTNR really not need .moas.gl. in the parser name?

Reply:
The glider NUTNR driver is here: https://github.com/oceanobservatories/mi-instrument/tree/master/mi/dataset/driver/nutnr_m/glider while the rest of the glider drivers are here: https://github.com/oceanobservatories/mi-instrument/tree/master/mi/dataset/driver/moas/gl

I'm not sure if there was a reason it was kept separate from the rest, but an argument could definitely be made that it should be moved.